x-pack/plugins/security_solution/cypress/e2e/entity_analytics/entity_analytics_management_page.cy.ts

```diff
@@ -0,0 +1,119 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import {
+  PAGE_TITLE,
+  HOST_RISK_PREVIEW_TABLE,
+  HOST_RISK_PREVIEW_TABLE_ROWS,
+  USER_RISK_PREVIEW_TABLE,
+  USER_RISK_PREVIEW_TABLE_ROWS,
+  RISK_PREVIEW_ERROR,
+  RISK_PREVIEW_ERROR_BUTTON,
+} from '../../screens/entity_analytics_management';
+
+import { login, visit, visitWithoutDateRange } from '../../tasks/login';
+import { cleanKibana } from '../../tasks/common';
+import { ENTITY_ANALYTICS_MANAGEMENT_URL, ALERTS_URL } from '../../urls/navigation';
+import { esArchiverLoad, esArchiverUnload, esArchiverResetKibana } from '../../tasks/es_archiver';
+import { getNewRule } from '../../objects/rule';
+import { createRule } from '../../tasks/api_calls/rules';
+import { waitForAlertsToPopulate } from '../../tasks/create_new_rule';
+import {
+  SHOW_DATES_BUTTON,
+  DATE_PICKER_ABSOLUTE_TAB,
+  DATE_PICKER_ABSOLUTE_INPUT,
+  LOCAL_DATE_PICKER_START_DATE_POPOVER_BUTTON,
+  LOCAL_DATE_PICKER_END_DATE_POPOVER_BUTTON,
+  LOCAL_DATE_PICKER_APPLY_BUTTON_TIMELINE,
+} from '../../screens/date_picker';
+
+describe(
+  'Entity analytics management page',
+  { env: { ftrConfig: { enableExperimental: ['riskScoringRoutesEnabled'] } } },
+  () => {
+    before(() => {
+      esArchiverResetKibana();
+      cleanKibana();
+    });
+
+    beforeEach(() => {
+      login();
+      esArchiverLoad('all_users');
+      visitWithoutDateRange(ALERTS_URL);
+      createRule(getNewRule({ query: 'user.name:* or host.name:*', risk_score: 70 }));
+      waitForAlertsToPopulate();
+      visit(ENTITY_ANALYTICS_MANAGEMENT_URL);
+    });
+
+    after(() => {
+      esArchiverUnload('all_users');
+    });
+
+    it('renders page as expected', () => {
+      cy.get(`${PAGE_TITLE}`).should('have.text', 'Entity Risk Score');
+    });
+
+    describe('Risk preview', () => {
+      it('there is data for risk and host preview react on date range', () => {
```

Comment:
In this test we are checking also what happens when there are no items, so I would split it since the name of the test is not reflecting what is actually doing.